frontend/views/components/VotingRulesInput.vue

```diff
@@ -46,7 +46,7 @@ export default ({
   data: () => ({
     config: {
       [RULE_PERCENTAGE]: {
-        sliderLabel: L('What percentage of members need to agree?'),
+        sliderLabel: L('What percentage of members need to agree to a proposal?'),
```

Comment:
One minor change request here (I made the same change request on Figma just now for @leihla as well): can we change this to:

> What percentage of members need to agree to pass a proposal?